src/components/analyses/listing/Listing.js

```diff
@@ -374,12 +396,13 @@ function Listing(props) {
                 onDetailsSelected={onDetailsSelected}
                 handleInteractiveUrlClick={handleInteractiveUrlClick}
                 handleGoToOutputFolder={handleGoToOutputFolder}
+                handleDelete={handleDelete}
                 handleRelaunch={handleRelaunch}
                 handleBatchIconClick={handleBatchIconClick}
             />
             <TableView
-                loading={isFetching || relaunchLoading}
-                error={error || relaunchError}
+                loading={isFetching || deleteLoading || relaunchLoading}
+                error={error || deleteError || relaunchError}
```

Comment:
Sorry I missed this during `relaunch` PR but `errors` which happen during mutations should be handled with `announcer`. Blocking the listing will prevent the users from retrying these operations.
 I think we agreed during our alpha test review meeting that 
1. Any listing error should be redirected to `error.js` page.Users can then simply hit browser `Back` button to retry the listing.
2. Any other error should be handled by displaying an `announcer` with ` Details` button if applicable.